src/applications/vaos/referral-appointments/CompleteReferral.jsx

```diff
@@ -79,8 +79,8 @@ export default function CompleteReferral() {
         hasEyebrow
         heading={
           appointmentInfoTimeout
-            ? "We're having trouble scheduling this appointment"
-            : "We can't schedule this appointment online"
+            ? 'We’re having trouble scheduling this appointment'
```

Comment:
Note: Some changes from straight apostrophe to curvy which is what VA wants. Another ticket will go through and change any others.